resources/js/pages/Tool.vue

```diff
@@ -88,22 +88,22 @@
             />
 
             <div class="w-full md:w-3/5 md:px-3 mb-2 md:mb-0">
-                <input type="text" v-model="customCommand.command" placeholder="Enter a Command..." class="w-full form-control form-input form-input-bordered">
+                <input type="text" v-model="customCommand.command" :placeholder="this.__('Enter a Command...')" class="w-full form-control form-input form-input-bordered">
```

Comment:
Is there a particular reason for using `this` in the template? I think it should be removed.
```suggestion
                <input type="text" v-model="customCommand.command" :placeholder="__('Enter a Command...')" class="w-full form-control form-input form-input-bordered">
```